Resources/Prototypes/Catalog/Fills/Backpacks/StarterGear/backpack.yml

```
@@ -252,6 +260,10 @@
         - id: EpinephrineChemistryBottle
         - id: EpinephrineChemistryBottle
         - id: WeaponEnergyGunPistol
+      # PIRATE START
+        - id: EnergySwordHoS
+      # PIRATE END
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)